source/partials/release_notes/_release-20-3-0.md.erb

```diff
@@ -0,0 +1,67 @@
+<h4>New Pluggable SCMs Page</h4>
+
+We have introduced a new Pluggable SCMs page to create and manage Plugin Materials.
+
+<figure class='shrink-figure-100'>
+    <%= image_tag 'releases/20.3.0/pluggable_scm_spa.png', alt: 'Pluggable SCMs page' %>
+</figure>
+
+<h4>UI/UX refresh</h4>
+
+As part of our ongoing UI/UX refresh for improving the user experience, we have rewritten the Packages page to keep it in sync with other pages and retaining the existing functionality.
+
+<h4>New Packages Page</h4>
+
+<figure class='shrink-figure-100'>
+    <%= image_tag 'releases/20.3.0/package_repository_spa.png', alt: 'Package Repository page' %>
+</figure>
+
+<h4>Comparison Page</h4>
+
+<figure class='shrink-figure-100'>
+    <%= image_tag 'releases/20.3.0/comparison_spa.png', alt: 'Comparison page' %>
+</figure>
+
+<h4>Other Improvements</h4>
+
+* <%= link_to_issue 7860, 'Pipelines page: disable delete for pipelines with dependent pipelines' %>
+* <%= link_to_issue 7889, '[Performace Improvement] Remove unnecessary group by cause while loading dashboard pipeline' %>
+* <%= link_to_issue 8021, 'New Pipeline wizard supports selecting Packages and Pluggable SCM Materials' %>
```

Comment:
Also related to https://github.com/gocd/gocd/issues/2947 and https://github.com/gocd/gocd/issues/7022 (in case we want to mention).

Reply:
Will update the PR to add those as well